ros2bag/ros2bag/verb/play.py

```diff
@@ -60,18 +60,24 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         parser.add_argument(
             '--services', type=str, default=[], metavar='service', nargs='+',
             help='Space-delimited list of services to play.')
+        parser.add_argument(
+            '--actions', type=str, default=[], metavar='action', nargs='+',
+            help='Space-delimited list of actions to play.')
         parser.add_argument(
             '-e', '--regex', default='',
-            help='Play only topics and services matches with regular expression.')
+            help='Play only topics, services and actions matches with regular expression.')
         parser.add_argument(
             '-x', '--exclude-regex', default='',
-            help='regular expressions to exclude topics and services from replay.')
+            help='regular expressions to exclude topics, services and actions from replay.')
         parser.add_argument(
             '--exclude-topics', type=str, default=[], metavar='topic', nargs='+',
             help='Space-delimited list of topics not to play.')
         parser.add_argument(
             '--exclude-services', type=str, default=[], metavar='service', nargs='+',
             help='Space-delimited list of services not to play.')
+        parser.add_argument(
+            '--exclude-actions', type=str, default=[], metavar='action', nargs='+',
+            help='Space-delimited list of actions not to play.')
```

Comment:
test would be ideal for these additional arguments to make sure that does not break cli.

Reply:
Currently, this play command is not tested directly; instead, the tests are placed where the filter actually works, such as in test_mcap_topic_filter.cpp and test_sqlite_topic_filter.cpp.

Reply:
Just like ros2bag/test/test_recorder_args_parser.py, a test file for play_args needs to be added.